swift/Sources/Blockchains/Stellar.swift

```diff
@@ -12,32 +12,3 @@ public typealias TWStellarMemoText = TW_Stellar_Proto_MemoText
 public typealias TWStellarMemoVoid = TW_Stellar_Proto_MemoVoid
```

Comment:
do we still needed `typealias` ? can we generate them or have in one class?

Reply:
Yes, there is an issue for that